src/application.rs

```diff
@@ -236,38 +269,24 @@ pub trait ApplicationHandler {
     /// destroyed, which indirectly invalidates any existing render surfaces that may have been
     /// created outside of Winit (such as an `EGLSurface`, [`VkSurfaceKHR`] or [`wgpu::Surface`]).
     ///
-    /// After being `Suspended` on Android applications must drop all render surfaces before
+    /// After being [suspended] on Android applications must drop all render surfaces before
```

Comment:
Same here, we probably shouldn't maintain the `suspended` wording for simplicity and to reduce complexity. Let the users' `Surface`, `Swapchain` etc lifetimes purely depend on the two `surface()` callbacks introduced here, and nothing else.

Reply:
Addressed in #3786.